app/views/AddApplicationSubordinateSuccessView.scala.html

```diff
@@ -23,7 +23,10 @@
 
 @(appName: String, id: ApplicationId)(implicit request: play.api.mvc.Request[Any], loggedIn: DeveloperSession, messages: Messages, appConfig: config.ApplicationConfig)
 
-@pageHeading = @{"Application added to the sandbox"}
+@theSandbox = @{"the sandbox"}
```

Comment:
Lots of repeats of this style.  Could we reduce this ?

Reply:
Fixed in latest version. Now lives in an EnvironmentNameService.